modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -35,9 +35,11 @@ mp1       No            2           m5.xlarge                        us-east-1a
 +
 [source,terminal]
 ----
-$ rosa edit machinepool --cluster=<cluster_name> --replicas=<number_worker_nodes> <machinepool_ID>
+$ rosa edit machinepool --cluster=<cluster_name> --replicas=<number_worker_nodes> <machine_pool_id>
```

Comment:
Should we clarify that autoscaling machinepool must add --enable-autoscaling=false once user want to update it to the one with replicas?

Reply:
@xueli181114 thank you for this suggestion. On review, I think that might be better positioned in the documentation on autoscaling. I think in this assembly it is useful to focus on what is possible around manual scaling of machine pools that don't use autoscaling, rather than how to switch between the two modes.

I have updated the heading for the module to "Scaling compute nodes manually" and stated the purpose of the section explicitly in the introduction:

> If you have not enabled autoscaling for your machine pool, you can manually scale the number of compute (also known as worker) nodes in the pool to meet your deployment needs.